lib/solidus_stripe/webhook/event.rb

```diff
@@ -24,7 +24,7 @@ class Event
       # @api private
       class << self
         def from_request(payload:, signature_header:, slug:, tolerance: default_tolerance)
-          payment_method = SolidusStripe::WebhookEndpoint.payment_method(slug)
+          payment_method = SolidusStripe::PaymentMethod.with_slug(slug).first!
```

Comment:
Not sure to get why we unwrapped the procedure to get the payment method. IMHO, that opens gates to duplication and copy-paste programming.

Reply:
@waiting-for-dev that's a valid concern. The reason it was moved to a slightly lower-level construct like a scope is that it's more versatile and can be mixed and matched with other scopes without generating extra queries more easily, e.g. in [this case](https://github.com/solidusio/solidus_stripe/pull/271/files/731629d88e5e2dea570686dd412a45b0fd77cb30#diff-731d975f879e89e2da5e248ccdfdcf53d9aebf9a8d5add2289218a32f23b2627R47)

https://github.com/solidusio/solidus_stripe/blob/731629d88e5e2dea570686dd412a45b0fd77cb30/app/controllers/solidus_stripe/intents_controller.rb#L45-L48